tests/server/conftest.py

```diff
@@ -1,20 +1,53 @@
 """Test fixtures for cg/server tests"""
 import os
+from typing import Generator
 
 import pytest
+from flask import Flask
+from flask.testing import FlaskClient
+from mock import patch
 
-from cg.server.app import create_app
+from cg.constants import DataDelivery, Pipeline
+from cg.store.database import create_all_tables, drop_all_tables
+from cg.store.models import Case
+from tests.store_helpers import StoreHelpers
 
-os.environ["CG_SQL_DATABASE_URI"] = "sqlite:///:memory:"
+os.environ["CG_SQL_DATABASE_URI"] = "sqlite:///"
 os.environ["LIMS_HOST"] = "dummy_value"
 os.environ["LIMS_USERNAME"] = "dummy_value"
 os.environ["LIMS_PASSWORD"] = "dummy_value"
 os.environ["GOOGLE_OAUTH_CLIENT_ID"] = "dummy_value"
 os.environ["GOOGLE_OAUTH_CLIENT_SECRET"] = "dummy_value"
+os.environ["CG_ENABLE_ADMIN"] = "1"
 
 
 @pytest.fixture
-def app():
-    """Test fixture to use when the flask app is needed"""
-    _app = create_app()
-    return _app
+def app() -> Generator[Flask, None, None]:
+    from cg.server.auto import app
+
+    app.config.update({"TESTING": True})
+    create_all_tables()
+    yield app
+    drop_all_tables()
+
+
+@pytest.fixture
+def case(helpers: StoreHelpers) -> Case:
+    from cg.server.ext import db as store
```

Comment:
Is importing inside a fixture a common practice? Why not import the module outside?

Reply:
It was a necessary workaround since we create the flask app which instantiates the store. I agree it can be improved, but we did not find a better way to do it

Reply:
Moving it outside works fine, good catch 👍